pyflakes/checker.py

```diff
@@ -739,6 +739,10 @@ def _module_scope_is_typing(name):
             isinstance(node.value, ast.Name) and
             _module_scope_is_typing(node.value.id) and
             is_name_match_fn(node.attr)
+        ) or (
+            isinstance(node, ast.Constant) and
```

Comment:
fyi, mypy understands `x: 'TypeAlias' = Foo` (https://mypy-play.net/?mypy=latest&python=3.10&gist=f7ee6de9eea2f197d6c43039acd97ead)

although I didn't see this in the spec https://www.python.org/dev/peps/pep-0613/ I thought it could be supported also

Reply:
let's skip that complexity for now -- it seems very unlikely that someone would do that